java/test/src/main/java/io/ray/test/MetricTest.java

```diff
@@ -20,6 +23,55 @@ boolean doubleEqual(double value, double other) {
     return value <= other + 1e-5 && value >= other - 1e-5;
   }
 
+  private MetricConfig initRayMetrics(long timeIntervalMs,
```

Comment:
Shutdown registy after method is finished.

Reply:
Fixed